src/Type/Definition/DescriptionAwareInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Digia\GraphQL\Type\Definition;
+
+interface DescriptionAwareInterface
+{
+    /**
+     * @return null|string
+     */
+    public function getDescription(): ?string;
+}
```

Comment:
Introduce this interface to be able to check whether or not the class has a description.